pages/vi/profiles/inDepthh.md

```diff
@@ -0,0 +1,26 @@
+# Hi,
+
+### My name is Ryan Couture, I live in the United States of America (Eastern Standard time - UTC-05:00)
+
+
+*OS: Windows 10 Pro Version 1803*
+
+	  Description: I am a passionate Software Development student at NHTI.
+	  I am graduating soon and hope to find a job after. This internship will
+	  give me a lot of valuable experience to get a job in IT.```
+
+**I have good experience with Languages such as**
+* Java
```

Comment:
looks like you wanted to add bullet here but its not adding bullet

Reply:
Yes, doesn't make a lot of sense to me. On the preview tab it works perfectly, but on raw.githack it did not process it properly. I also tested it on a 3rd party service called StackEdit, and it worked well.

![file-indepthh md](https://user-images.githubusercontent.com/20867959/47740036-29ff7480-dc4d-11e8-9e20-b7e88a7b7992.png)